test/functional/insert.test.js

```diff
@@ -1018,7 +1102,9 @@ describe('Insert', function() {
     // Add a tag that our runner can trigger on
     // in this case we are setting that node needs to be higher than 0.10.X to run
     metadata: {
-      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
+      requires: {
```

Comment:
what's motivating these whitespace changes? Is your `eslint` telling you the lines need to be fixed?

Reply:
I have no idea... I'm reverting and rerunning prettier and eslint and its leaving it the old way, should've noticed these unnecessary changes, will revert.